app/Console/Commands/TagPosts.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Tag;
+use Rogue\Models\Post;
+use Illuminate\Console\Command;
+
+class TagPosts extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:tagposts {--tag=: The id of the tag to use.} {--posts=: Comma-separated list of the post_ids to update.}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Tag a set of posts with the given tag id';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $postIds = explode(',', $this->option('posts'));
+        $posts = Post::find($postIds);
+
+        $tag = Tag::findOrFail($this->option('tag'));
+
+        if ($posts) {
+            foreach ($posts as $post) {
+                $this->info('Tagging post_id: '.$post->id.' as '.$tag->tag_slug);
+                $post->tag($tag->tag_name);
```

Comment:
If the post has already been tagged with that particular tag, will this untag it?

Reply:
Nope! The logic just attaches the relationship and duplicates are not allowed in the table: https://github.com/DoSomething/rogue/blob/master/app/Models/Post.php#L242-L254

Reply:
@sbsmith86 cool! So since this command updates the Posts directly without passing through the PostService, I think we will want to fire off a `SendPostToQuasar` job here?

Reply:
@sbsmith86 given that we are already not sending the tags over to Quasar (🐛 ) maybe we don't need to worry about this here?